pom.xml

```diff
@@ -34,7 +34,7 @@
     <inceptionYear>2019</inceptionYear>
 
     <properties>
-        <easy.schema.version>3.0.2</easy.schema.version>
+        <easy.schema.version>3.1.1-SNAPSHOT</easy.schema.version>
```

Comment:
@janvanmansum not sure what version you want to use after splitting `easy-schema`. Please wait with merging this PR until we have created the release of that project.